BDArmory/BDArmory.csproj

```diff
@@ -949,4 +950,7 @@
     <PreBuildEvent>
     </PreBuildEvent>
   </PropertyGroup>
+  <PropertyGroup>
```

Comment:
These 3 lines override the OS specific post-build events on lines 856—926.
I'm not sure if the flags are the same on Windows, but in Linux if you want to skip the post-build events, you can build it with
```bash
dotnet build --configuration Debug /p:PostBuildEvent=
```